subgraph/.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules
+data
+build
+generated
+
+subgraph.yaml
+bun.lockb
```

Comment:
We don't have `bun.lockb` files and let's keep new line at the end of the file.

Reply:
noted with thanks